src/app/login/login.component.ts

```diff
@@ -0,0 +1,28 @@
+import { Component } from '@angular/core';
+import { AmplifyService } from 'aws-amplify-angular';
+
+@Component({
+  selector: 'ata-login',
+  templateUrl: 'login.component.html',
+  styleUrls: ['login.component.css']
+})
+export class LoginComponent {
+  signedIn: boolean;
```

Comment:
Not sure how that amplify service works, but if authState.user is null when 'signedIn' is false, you could do something like this.
```suggestion
  signedIn$ = this.user$.pipe(map(user => !!user));
  user$ = this.amplifyService.authStateChange$.pipe(
    map(({ user }) => user)
  )
```

Reply:
signedIn: boolean; 
checks the authStatChange$ if or else statement for authState.state which is apart of AmplifyService.
Looks like when a user signs in the boolean is switch to true which displaying the greeting message. Might need to get your eye on this